app/src/main/java/org/gdg/frisbee/android/api/PlusImageUrlConverter.java

```diff
@@ -0,0 +1,41 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.api.model.plus.ImageInfo;
+import org.gdg.frisbee.android.app.App;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class PlusImageUrlConverter implements Interceptor {
```

Comment:
Let's do that in a different PR
On 18 Mar 2016 17:08, "Said Tahsin Dane" <notifications@github.com> wrote:

> In
> app/src/main/java/org/gdg/frisbee/android/api/PlusImageUrlConverter.java
> <https://github.com/gdg-x/frisbee/pull/589#discussion_r56680179>:
>
> > @@ -0,0 +1,41 @@
> > +package org.gdg.frisbee.android.api;
> > +
> > +import org.gdg.frisbee.android.api.model.plus.ImageInfo;
> > +import org.gdg.frisbee.android.app.App;
> > +
> > +import java.io.IOException;
> > +import java.util.regex.Matcher;
> > +import java.util.regex.Pattern;
> > +
> > +import okhttp3.Interceptor;
> > +import okhttp3.Request;
> > +import okhttp3.Response;
> > +
> > +public class PlusImageUrlConverter implements Interceptor {
>
> http://gde-map.appspot.com/gde/list
>
> now returns linkedin urls for social accounts instead of Google+. That's
> definitely a bug but we cannot trust it anymore. I suggest that we can
> disable GDE feature for this release. We can remove this ugly interceptor
> completely.
>
> And we can re-write GDE API later.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/589/files/0260fb0df84af023e555b14602ed23796d885a5f#r56680179>
>